src/main/java/cinema/security/UserPrincipal.java

```diff
@@ -0,0 +1,57 @@
+package cinema.security;
+
+import cinema.model.User;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+public class UserPrincipal implements UserDetails {
```

Comment:
let's remove this class
and use `loadUserByUsername()` implementation given in the lesson materials

Reply:
done